packages/headache/headache.1.04/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+name: "headache"
+version: "1.04"
```

Comment:
name and version are redundant, the are inferred from the path (and overridden). I can remove them before merging, unless you need to make further changes to the opam file

Reply:
Thanks for the proposal. I will leave you remove such lines once the automatic checks will be ok.
Could you also have a look to the `dev-repo` field ?

Reply:
Fields `name:` and `version:` have been removed (see commit  4c202b3)